koku/reporting/migrations/0031_ocpawscostlineitemdailysummary_currency_code.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1.5 on 2019-02-13 19:41
```

Comment:
We shouldn't be altering already committed migrations like this. They are a record of changes made to the database and if they don't reflect what actually happened we can be out of sync, or our dev database build migrations might end up different from production.

Reply:
Don't worry about the migrations, the end result is fine. so ignore this one.